### PR TITLE
chore(deps): Upgrade typings

### PR DESCRIPTION
**Pull request by sirreal**
#### Changes proposed in this Pull Request

Upgrade and deduplicate @types and csstype packages.

#### Testing instructions

CI passing should be sufficient.

### CODE REVIEW

**Comment by scinos** [comment]
There are many changes in `yarn.lock` that are not about types. For example, `@wordpress/element@^2.14.0` was resolved as `2.14.0` before, but it is `2.18.0` now.

Are those expected?

**Comment by sirreal** [comment]
> There are many changes in `yarn.lock` that are not about types. For example, `@wordpress/element@^2.14.0` was resolved as `2.14.0` before, but it is `2.18.0` now.
> 
> Are those expected?

Absolutely not, thanks for catching it!

It seems like new dependencies combined with `*` dependencies and, although the existing dependencies satisfied the new requirement, triggered the addition of an upgraded package.

I've manually done a bunch of updating and this seems to be in the correct state now with desired `@types/*` packages upgraded.

File: packages/viewport-react/package.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
 		"@wordpress/compose": "^3.19.3"
 	},
 	"peerDependencies": {
-		"react": "^16.13.1",
+		"react": "^16.12.0",
```
Comment (scinos) [inline packages/viewport-react/package.json]: Why this downgrade?

Comment (sirreal) [reply packages/viewport-react/package.json]: It's probably not necessary, but I was surprised how this seems to be the package the makes 16.13.1 be the version for the entire repo and went for consistency. Should be reverted.
```suggestion
		"react": "^16.13.1",
```

**Comment by sirreal** [comment]
Ok, I think this is in the right place now. Through upgrades and deduplication I managed to get to what appears to be safe, @types-only upgrades. Review would be appreciated at this time.

File: yarn.lock [hunk 2]
```diff
@@ -4079,12 +4039,7 @@
     "@types/node" "*"
     form-data "^3.0.0"
 
-"@types/node@*", "@types/node@>= 8":
-  version "13.7.4"
-  resolved "https://registry.yarnpkg.com/@types/node/-/node-13.7.4.tgz#76c3cb3a12909510f52e5dc04a6298cdf9504ffd"
-  integrity sha512-oVeL12C6gQS/GAExndigSaLxTrKpQPxewx9bOcwfvJiJge4rr7wNaph4J+ns5hrmIV2as5qxqN8YKthn9qh0jw==
-
-"@types/node@^13.7.0":
+"@types/node@*", "@types/node@>= 8", "@types/node@^13.7.0":
```
Comment (scinos) [inline yarn.lock:4042]: I don't know if this is important or if we can even do anything about it... but having types for Node 13.x when we are running Node 12.x is weird.

**Review by scinos — APPROVED** [review]

File: yarn.lock (hunk 2 above)
Comment (sirreal) [reply yarn.lock:4042]: Agreed, but a quick scan suggests we don't depend on it directly so… 🤷

**Pull request merged**